fcrepo-jms/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>javax.inject</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.modeshape</groupId>
```

Comment:
Yes, it is the JCR_CONTENT constant. It implies a deeper investigation as to why it is escaping the fcrepo-kernel-modeshape module. And yes, it is most certainly a good "next target".